package.json

```diff
@@ -1937,6 +1937,15 @@
                     "application/vnd.jupyter.widget-view+json"
                 ]
             },
+            {
+                "id": "jupyter-error-renderer",
+                "entrypoint": "./out/datascience-ui/errorRenderer/errorRenderer.js",
+                "displayName": "Jupyter Error Renderer",
+                "requiresMessaging": "always",
+                "mimeTypes": [
+                    "application/vnd.code.notebook.error"
```

Comment:
What will happen when irun a Jupyter notebook? Will the hyperlinks for errors work there?
I don't think it will, shouldn't we disable the links or get it working, else it's weird to have links and do nothing.
If it works or doesn't have links for error lines then fine, else I think it's a problem.

Reply:
> I think we should consider what we do when in notebooks. Not a fan of it not working in notebook output.

It works in notebook, if `preHandleIOPub` is for all kernel IOs, not just interactive window.

And also if the outputs don't have a `outputItem.metadata.transient: string[]`, we will use the raw error data directly, so it has a default fall back. But we may want to give it a better namespace, like `outputItem.metadata.transient.modifiedTraceback`.

Reply:
good catch though, the prehandleoppub code is not robust enough that it doesn't fail gracefully when it fails to find the file link

what it should render:

```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-2-659abc318634> in <module>
----> 1 print(fo)

NameError: name 'fo' is not defined
```

what it renders:

```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
 in 
----> 1 print(fo)

NameError: name 'fo' is not defined
```